setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='edx-lint',
-    version='0.3.0',
+    version='0.3.2',
```

Comment:
Why the bump from `0.3.0` to `0.3.2`, rather than `0.3.1`? Not a problem, just curious for the rationale.